### PR TITLE
support fit push

### PR DESCRIPTION
**Pull request by saffi-codefresh**

### CODE REVIEW

**Review by itai-codefresh — CHANGES_REQUESTED** [review]
please create a new version and don't override existing 0.0.1

File: workflows/git/versions/0.0.1/README.md [hunk 1]
```diff
@@ -6,10 +6,129 @@ A set of generics git operations
...
 
 ## Security
 
 Minimal required permissions
 
 [Full rbac permissions list](https://github.com/codefresh-io/argo-hub/blob/main/workflows/git/versions/0.0.1/rbac.yaml)
+
+
+## Examples
+### Clone change commit scenario
```
Comment (itai-codefresh) [inline workflows/git/versions/0.0.1/README.md]: lets be allgiend. we don't put examples in the main READ me. move them to the specific templates

**Review by itai-codefresh — CHANGES_REQUESTED** [review]

File: workflows/git/versions/0.0.1/README.md (hunk 1 above)
Comment (saffi-codefresh) [reply workflows/git/versions/0.0.1/README.md]: Boss. 
a. moved to specific templates
b. I have example which shows how the steps works together, Please advice where to place it.

Comment (saffi-codefresh) [reply workflows/git/versions/0.0.1/README.md]: moved into the commit template example

**Review by itai-codefresh — APPROVED** [review]

**Pull request merged**